torch/optim/lr_scheduler.py

```diff
@@ -173,14 +208,96 @@ def load_state_dict(self, state_dict):
                 self.lr_lambdas[idx].__dict__.update(fn)
 
     def get_lr(self):
+        if not self._get_lr_called_within_step:
+            warnings.warn("To get the last learning rate computed by the scheduler, "
+                          "please use `get_last_lr()`.")
+
         return [base_lr * lmbda(self.last_epoch)
                 for lmbda, base_lr in zip(self.lr_lambdas, self.base_lrs)]
 
 
+class MultiplicativeLR(_LRScheduler):
```

Comment:
This could live in a separate PR, right?

Reply:
It could, yes. The reason I added it in is that LambdaLR felt a little out of place alone since it was not chainable. Do you recommend doing so?

Reply:
MultiplicativeLR moved to #27254